tests/inputs/res/assembling.xsd

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema version="1.0"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified">
+
+  <xs:element name="assembling">
+    <xs:complexType>
+      <xs:choice maxOccurs="unbounded">
+        <xs:element name="singleAssociation" type="SingleAssociation"/>
+        <xs:element name="dynamicAutomaton" type="GenericModel"/>
+        <xs:element name="multipleAssociation" type="MultipleAssociation"/>
```

Comment:
Would be nice to not duplicate this file

Reply:
I require a wrong setting.xsd file for my test and it was easier to copy the other one

Reply:
assembling.xsd removed